frontend/components/site/siteSettings.js

```diff
@@ -112,61 +112,53 @@ class SiteSettings extends React.Component {
         </div>
         <div className="usa-grid">
           <div className="usa-width-one-whole">
-            <div className="form-group">
-              <label className="active" htmlFor="domain">Custom domain</label>
-              <input
-                name="domain"
-                className="form-control"
-                type="text"
-                placeholder="https://example.com"
-                value={ state.domain }
-                onChange={this.onChange} />
-            </div>
-            <div className="usa-alert usa-alert-info">
-              <div className="usa-alert-body">
-                <h3 className="usa-alert-heading">Custom Domain</h3>
-                <p className="usa-alert-text">This read-only origin box is used for Federalist to set a custom domain on the 18F side. Federalist customers only need this information if handling custom URL work directly.</p>
-                <input readOnly type="text" value="" />
+            <div className="form-group">              
+              <div className="usa-alert usa-alert-info">
+                <div className="usa-alert-body">
+                  <h3 className="usa-alert-heading">Custom Domain</h3>
+                  <p className="usa-alert-text">If you build your site with Jekyll, Federalist can configure your production site to load at a custom domain specified here.</p>
+                  <input
+                   name="custom-domain"
+                   className="form-control"
+                   type="text"
+                   placeholder="https://example.com"
+                   value={ state.domain }
+                   onChange={this.onChange} />
+                </div>
               </div>
             </div>
           </div>
         </div>
         <div className="usa-grid">
           <div className="usa-width-one-whole">
-            <div className="form-group">
-              <label htmlFor="config" className="">Site custom configuration</label>
-              <textarea
-                name="config"
-                className="form-control"
-                value={state.config}
-                onChange={this.onChange} />
-            </div>
             <div className="usa-alert usa-alert-info">
               <div className="usa-alert-body">
                 <h3 className="usa-alert-heading">Site configuration</h3>
                 <p className="usa-alert-text">
                   Add additional configuration in yaml to be added to your <code>_config.yml</code> file when we build your site's default branch.
                 </p>
```

Comment:
I believe the angry red color is the result of GitHub trying to parse the JSX as JS.